### PR TITLE
UI: [VAULT-17365, VAULT-17613, VAULT-17614] Dashboard Secrets Engine Card Improvements

### PR DESCRIPTION
**Pull request by kiannaquach**
**Description**
Some small improvements to secret engine card based on design feedback
- VAULT-17365 round secret engine table
- VAULT-17613 truncate lines
- VAULT-17614 Add bold + underline
Screenshot 2023-07-06 at 9 59 30 AM

### CODE REVIEW

File: ui/app/styles/core.scss [hunk 1]
```diff
@@ -62,6 +62,7 @@
 @import './components/confirm';
 @import './components/console-ui-panel';
 @import './components/control-group';
+@import './components/dashboard';
```
Comment (Monkeychip) [inline ui/app/styles/core.scss:65]: so technically these are only meant to correspond to a component called "dashboard". And it looks like you're using a route called Dashboard. 

Maybe you and I can sync real quick on this. Ideally (I know you mentioned above about it not being a helper class) but it really doesn't make sense as a component class.

**Review by Monkeychip — COMMENTED** [review]
Would love to try some things first before adding the component class file. Let's sync when you get a moment.

File: ui/app/styles/core.scss (hunk 1 above)
Comment (kiannaquach) [reply ui/app/styles/core.scss]: synced with angel and will move dashboard route template to a component file if extra styling is needed!

**Review by Monkeychip — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**